### ISSUE TITLE
Proposal: Quick patch to use edit action when option key pressed.

### ISSUE THREAD

**Issue by mikeerickson** [dcasia/clickable-table-row#7]
I just made a quick patch to the source to allow using `option` key to perform `edit` action instead of always perform `view` action

On line 18 of `tools.js file`, I changed to 

```js
     let selector = 'a[dusk$="-' + (event.altKey ? 'edit' : 'view') + '-button"]'
     const viewElement = this.querySelector(selector)
```

If you have the desire to allow this as PR, let me know and I will get it submitted.  For now, it does exactly as I want so I am good either way :-)

**Comment by milewski** [comment]
Please send a PR I will merge

**Comment by mikeerickson** [comment]
@milewski Have a look at this PR https://github.com/dcasia/clickable-table-row/pull/8 which add support for using the `option` key to display `edit` view

**Comment by milewski** [comment]
@mikeerickson thanks! released on `v0.1.5`
